### PR TITLE
[Merged by Bors] - [cu-206yhnr] Refactoring build process and imports for Composable JS.

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
Previously there was a complicated typescript configuration for each package which wasn't needed

## Description
- Each package acts as an independent project
- tsconfig separate within all packages
- remove unncessary index from packages folder
- added `bootstrap_pallets` package

### CODE REVIEW

File: composablejs/packages/oracle_setup/src/index.ts [hunk 1]
```diff
@@ -1,90 +1 @@
...
-main()
-  .then(() => {
-    console.log("Finished setting up oracle!");
-    process.exit(0);
-  })
-  .catch(err => {
-    console.error(err.toString());
-    process.exit(1);
-  });
+export * from "./handlers";
```
Comment (Dom-Roth) [inline composablejs/packages/oracle_setup/src/index.ts:1]: Don't know if we need to export the modules of the oracle setup script. 
It's not really intended to be used by other packages.

File: composablejs/package.json [hunk 2]
```diff
@@ -1,24 +1,8 @@
...
-      "./dist/packages/utils/dist/**/*.js",
-      "./dist/packages/utils/dist/**/*.d.ts"
-    ],
-    "./oracle_setup/*": [
-      "./dist/packages/oracle_setup/dist/**/*.js",
-      "./dist/packages/oracle_setup/dist/**/*.d.ts"
-    ]
-  },
   "scripts": {
     "build": "tsc -b -f tsconfig.json",
```
Comment (Dom-Roth) [inline composablejs/package.json:7]:
```suggestion
    "build": "tsc -b -f tsconfig-base.json",
```

Comment (Dom-Roth) [reply composablejs/package.json:7]: Actually, the `build` command doesn't work with the refactoring but we can still compile the whole package using lerna.
So i guess we can get rid of that command. (Note: Update `Makefile` as well.)

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-206yhnr Composable.js Integration](https://app.clickup.com/t/206yhnr)

**Review by Dom-Roth — APPROVED** [review]
LGTM

**Review by vimukthi-git — APPROVED** [review]
LGTM

**Pull request closed without merging**